spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe "projects show page", type: :feature do 
+  before(:each) do
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+    
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+    
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")    
+
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    @jay.projects << @boardfit
+    @jay.projects << @upholstery_tux
+    @erin.projects << @news_chic
+    @erin.projects << @lit_fit
+  end
+
+  it 'shows names of all contestants and their projects' do
+    visit "/contestants"
+
+    expect(page).to have_content(@jay.name)
+    expect(page).to have_content("Projects: #{@boardfit.name}, #{@upholstery_tux.name}")
+    expect(page).to have_content(@gretchen.name)
+    expect(page).to have_content(@kentaro.name)
+    expect(page).to have_content(@erin.name)
+    expect(page).to have_content("Projects: #{@news_chic.name}, #{@lit_fit.name}")
+  end
```

Comment:
I would use `within` blocks to better test where things should be showing up in your view. Here is an example broken up into two tests:
```ruby
it "I see a list of names of all the contestants" do
 visit "/contestants"

  expect(page).to have_content("Contestants")
   within "#contestants_list" do
    expect(page).to have_content(@jay.name)
    expect(page).to have_content(@gretchen.name)
    expect(page).to have_content(@kentaro.name)
    expect(page).to have_content(@erin.name)
   end
end 
```
then
```ruby
it "And under each contestant's name I see a list of the projects (names) that they've been on" do
  visit "/contestants"

  within "#contestants_list" do
    within "#contestant_#{@jay.id}" do
      expect(page).to have_content("News Chic")
      expect(page).to_not have_content("Upholstery Tuxedo")
      expect(page).to_not have_content("Boardfit")
      expect(page).to_not have_content("Litfit")
     end

     within "#contestant_#{@gretchen.id}" do
       expect(page).to have_content("News Chic and Upholstery Tuxedo")
       expect(page).to_not have_content("Boardfit")
       expect(page).to_not have_content("Litfit")
      end
      ....
   end
end
```
Then you would just need to add the css ids in your view to identify where those within blocks are pointing. It will look for the same name as you specify in `"within #nam"`
```ruby
<ul id="contestants_list">
  <% @contestants.each do |contestant| %>
  <li id="contestant_<%= contestant.id %>"><%= contestant.name %> <br/>
    Projects: <%= contestant.projects_list %>
  </li>
  <% end %>
</ul>
````